uaclient/apt.py

```diff
@@ -528,3 +528,22 @@ def setup_apt_proxy(
         util.remove_file(APT_PROXY_CONF_FILE)
     else:
         util.write_file(APT_PROXY_CONF_FILE, apt_proxy_config)
+
+
+def setup_unauthenticated_repo(
+    repo_filename: str,
+    repo_url: str,
+    keyring_file: str,
+    apt_origin: str,
+    suites: List[str],
+) -> None:
+    content = ""
+    for suite in suites:
+        content += messages.REPO_FILE_CONTENT.format(
+            url=repo_url, apt_suite=suite
+        )
+    util.write(repo_filename, content)
+    util.write()
```

Comment:
Why do we need the extra write here ?

Reply:
Oh, I was still brainstorming and failed to remove this write here.